docs/bucket/notifications/README.md

```diff
@@ -281,7 +287,11 @@ An example of Elasticsearch configuration is as follows:
 
 If Elasticsearch has authentication enabled, the credentials can be supplied to Minio via the `url` parameter formatted as `PROTO://USERNAME:PASSWORD@ELASTICSEARCH_HOST:PORT`.
 
-After updating the configuration file, restart the Minio server to put the changes into effect. The server will print a line like `SQS ARNs:  arn:minio:sqs::1:elasticsearch` at start-up if there were no errors.
+To update the configuration, use `mc admin config get` command to get the current configuration file for the minio cluster in json format, and save it locally.
+$ mc admin config get myminio/ > /tmp/myconfig
+
+After updating the Elasticsearch configuration in /tmp/myconfig , use `mc admin config set` command to update the configuration for the cluster.Restart the Minio server to put the changes into effect. The server will print a line like `SQS ARNs:  arn:minio:sqs::1:elasticsearch` at start-up if there were no errors.
+$ cat /tmp/myconfig | mc admin config set myminio/
```

Comment:
We can avoid `cat` instead say `mc admin config set myminio < /tmp/myconfig`  and change this everywhere.

Reply:
done.